dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESMappingAPIImpl.java

```diff
@@ -452,7 +455,7 @@ private void writeMetadata(final Contentlet contentlet, final StringWriter strin
 						mapLowered.put(compositeKey, value);
 
 						if (Config.getBooleanProperty(INDEX_DOTRAW_METADATA_FIELDS, true)
-								&& dotRawInclude.contains(metadataKey)) {
+								&& dotRawInclude.contains(metadataKey.toLowerCase())) {
```

Comment:
do we have a test for case sensitive keys?

Reply:
Test_toMap_Metadata_dotRaw() should take care of validating that what is expected on the index gets there